code/clue.js

```diff
@@ -3,33 +3,181 @@
 const mrGreen = {
   firstName: 'Jacob',
   lastName: 'Green',
-  color: 'green',
+  color: 'Green',
```

Comment:
`color` property of suspects objects should be written lowercase - CSS expect from us lowercase values of colours